src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -801,9 +801,9 @@ public boolean checkMachine_EM(IGregTechTileEntity iGregTechTileEntity, ItemStac
             return false;
         }
 
-        // Check if there is 1+ output bus, and they are ME output busses.
+        // Check if there is 1 output bus, and it is a ME output bus.
         {
-            if (mOutputBusses.size() == 0) {
+            if (mOutputBusses.size() != 1) {
```

Comment:
if you want to check weather its an ME input bus you will need to do a proper check like: mOutputBusses.get(0) instanceof OutputBus_ME ect